### PR TITLE
Preparation for Core release 1.32.0

### PR DESCRIPTION
**Pull request by annelo-msft**

### CODE REVIEW

File: sdk/core/Azure.Core/CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,12 @@
...
 
 - Added the `GetRawResponse` method to `RequestFailedException`.
 - Added overloads of `Operation<T>.WaitForCompletion` and `Operation.WaitForCompletionResponse` that take a `DelayStrategy`.
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added `DynamicData` type that provides a `dynamic` layer over raw JSON.  For more information, see [aka.ms/azsdk/net/dynamicdata](https://aka.ms/net/dynamicdata).
```
Comment (jsquire) [inline sdk/core/Azure.Core/CHANGELOG.md:9]:
```suggestion
- Added the `DynamicData` type that provides a `dynamic` layer over raw JSON.  For more information, see [aka.ms/azsdk/net/dynamicdata](https://aka.ms/net/dynamicdata).
```

File: sdk/core/Azure.Core/CHANGELOG.md [hunk 2]
```diff
@@ -31,13 +26,13 @@
 
 ### Bugs Fixed
 
-- Fixed the issue with empty header names and values, caused by `ArrayBackedPropertyBag` keeping reference to the array after returning it to array pool [in (https://github.com/Azure/azure-sdk-for-net/pull/34800)  `Dispose` method]. 
+- Fixed the issue with empty header names and values, caused by `ArrayBackedPropertyBag` keeping reference to the array after returning it to array pool [in (https://github.com/Azure/azure-sdk-for-net/pull/34800)  `Dispose` method].
```
Comment (jsquire) [inline sdk/core/Azure.Core/CHANGELOG.md:29]:
```suggestion
- Fixed an issue with empty header names and values, caused by `ArrayBackedPropertyBag` keeping reference to the array after returning it to array pool [in the `Dispose` method] (https://github.com/Azure/azure-sdk-for-net/pull/34800).
```

I *think* that was the link you intended?

**Review by jsquire — APPROVED** [review]

File: sdk/core/Azure.Core/CHANGELOG.md (hunk 1 above)
Comment (JoshLove-msft) [reply sdk/core/Azure.Core/CHANGELOG.md:9]: The URL doesn't seem to work.

Comment (JoshLove-msft) [reply sdk/core/Azure.Core/CHANGELOG.md:9]: I guess it is missing the azsdk path.

Comment (annelo-msft) [reply sdk/core/Azure.Core/CHANGELOG.md:9]: Good catch!

**Review by christothes — APPROVED** [review]

**Review by JoshLove-msft — APPROVED** [review]

**Pull request merged**